easybuild/easyconfigs/f/flex/flex-2.5.35-goolf-1.4.10.eb

```diff
@@ -8,7 +8,11 @@ sometimes called a tokenizer, is a program which recognizes lexical patterns in
 toolchain = {'name': 'goolf', 'version': '1.4.10'}
 toolchainopts = {'pic': True}
 
-sources = [SOURCELOWER_TAR_GZ]
-source_urls = ['http://prdownloads.sourceforge.net/%(namelower)s']
+sources = ['flex-2-5-35.tar.gz']
```

Comment:
@boegel note that in this particular case, that will also only work up to 2.5.36, after that they do use dots...